.changeset/fuzzy-swans-remain.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
+---
+
+GitHub issues label action:'github:issues:label' for Backstage plugin Scaffolder has been added.
```

Comment:
Maybe we can have something like this:

Added new `github:issues:label` action to apply labels to issues, and also output `pullRequestNumber` from `publish:github:pull-request`.